baseapp/baseapp.go

```diff
@@ -174,9 +181,8 @@ func (app *BaseApp) initFromStore(mainKey sdk.StoreKey) error {
 
 //----------------------------------------
 
-// Implements ABCI.
+// Info implements ABCI.
```

Comment:
Can we just keep these as before?  I know Golang recommends starting with the name of the method, but it seems silly.

Reply:
I agree in the code it seems silly, but I think go-lang recommends it in case people few it using godoc.

In godoc it reads nicer if you repeat the method/type name. [Example](https://godoc.org/bufio#Reader)

Reply:
Personally I don't care too much.

We should also keep in mind whether we want to run any linters.